### ISSUE TITLE
My Day Algorithms

### ISSUE THREAD

**Issue by medensqui** [PlanHubMe/PlanHub#30]
See My Day post

**Comment by thatoddmailbox** [comment]
@medensqui 
I guess we would need:
- Dividing homework into days
- Accounting for sleep (kind of same as above)
- After school commitment processing
- something else? @medensqui

**Comment by medensqui** [comment]
@thatoddmailbox 

No
No
Yes, (maybe a tab with suggested activities (i.e. Coffeehouse, ACC Teahouse, etc.)

I meant for priority for auto-scheduling and for taking away things when the person takes more time than estimated
Also maybe for estimating how long things will take based on tags?

**Comment by thatoddmailbox** [comment]
@medensqui why don't we need to divide into days?

yes that is an algorithm, though a simple one.

We could maybe show past times averaged per tag/subject. ("on average, you take bla amount of time for readings in history")

**Comment by willbarkoff** [comment]
We should ask user "How long do you think this assignment will take?" Then start estimating based on previous assignments from the same subject. I

**Comment by thatoddmailbox** [comment]
@nywillb Exactly!

> I 

?

I just found a bug in GitHub that allows me to edit anyones posts when I am not an owner. Any idea how to report bugs?

**Comment by willbarkoff** [comment]
@thatoddmailbox See your previous post.

**Comment by medensqui** [comment]
@thatoddmailbox @nywillb What do you guys think about having durations based on planner tags? Also, Studer, dividing into days while having this based around sleep doesn't work. Unless we would be able to subtract from one day and add to another day.

**Comment by willbarkoff** [comment]
@medensqui What do you mean by planner tags? Do you mean HTML tags or tags like organization. @medensqui @thatoddmailbox What if we have a slider for percentage complete?

**Comment by medensqui** [comment]
@nywillb No, because my idea was based around sleep and planning your day around how much sleep you get. No one gets enough sleep in the HS. It can be fully automated if you want. Percentage seems too much like My Study Life also.

**Comment by medensqui** [comment]
@nywillb Also write Quiz and then Test into planner view and see what happens. That's a tag

**Comment by medensqui** [comment]
@nywillb Or type Quiz and then 2.2

**Comment by willbarkoff** [comment]
@medensqui Oh, so you mean organazation tags.

**Comment by medensqui** [comment]
@nywillb Yes

**Comment by willbarkoff** [comment]
@medensqui Lets take a look at #44

**Comment by medensqui** [comment]
@nywillb Because?
